test/test.js

```diff
@@ -58,14 +58,19 @@ describe('transit', function() {
       d: '2'
     }, 'bar');
 
+    class BazRecord extends Immutable.Record({
```

Comment:
does this only happen with the class syntax? Could this be done using the older prototype approach which classes should just be sugar for?